<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgGuestEdit</class>
 <widget class="QDialog" name="dlgGuestEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="gbIdentification">
       <property name="title">
        <string> Identification </string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lblName">
            <property name="text">
             <string>Name :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ledName">
            <property name="minimumSize">
             <size>
              <width>340</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblGender">
            <property name="text">
             <string>Gender :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbGenderMale">
            <property name="text">
             <string>Male</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbGenderFemale">
            <property name="text">
             <string>Female</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lblUniqueId">
            <property name="text">
             <string>Unique identification :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ledUniqueId">
            <property name="minimumSize">
             <size>
              <width>211</width>
              <height>0</height>
             </size>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblDateBirth">
            <property name="text">
             <string>Date of birth :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="deDateBirth">
            <property name="dateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>1900</year>
              <month>1</month>
              <day>1</day>
             </datetime>
            </property>
            <property name="displayFormat">
             <string>yyyy/MM/dd</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblAge">
            <property name="text">
             <string>Age :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ledAge">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbPatientCard">
       <property name="title">
        <string> Guest's card </string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="lblBarcode">
          <property name="text">
           <string>Barcode :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledBarcode">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>170</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>170</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblPatientcardType">
          <property name="text">
           <string>Type of card :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledPatientcardType">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="pbSave">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
